<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCommandChangeExperienceCurve</class>
 <widget class="QDialog" name="DialogCommandChangeExperienceCurve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBoxSelection">
         <property name="title">
          <string>Selection</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0" colspan="3">
             <widget class="QRadioButton" name="radioButtonHeroEnemyInstanceID">
              <property name="text">
               <string>Hero / Enemy instance ID:</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="PanelPrimitiveValue" name="panelPrimitiveHeroEnemyInstanceID" native="true">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButtonEntire">
              <property name="text">
               <string>The entire</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBoxTeam">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2" colspan="3">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>298</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxLevel">
         <property name="title">
          <string>Level</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,1">
            <item>
             <widget class="QLabel" name="labelRange">
              <property name="text">
               <string>Range:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="PanelPrimitiveValue" name="panelPrimitiveRange" native="true">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelRangeTo">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>to</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="PanelPrimitiveValue" name="panelPrimitiveRangeTo" native="true">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxOperation">
         <property name="title">
          <string>Operation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="radioButtonEquals">
              <property name="text">
               <string>(=) Equals</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonPlus">
              <property name="text">
               <string>(+) Plus</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonMinus">
              <property name="text">
               <string>(-) Minus</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonTimes">
              <property name="text">
               <string>(*) Times</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonDivided">
              <property name="text">
               <string>(/) Divided by</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonModulo">
              <property name="text">
               <string>(%) modulo</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
         <item>
          <widget class="QLabel" name="labelTotalExperience">
           <property name="text">
            <string>Total experience:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PanelPrimitiveValue" name="panelPrimitiveTotalExperience" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCommandChangeExperienceCurve</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCommandChangeExperienceCurve</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
